src/components/Animate.js

```diff
@@ -0,0 +1,22 @@
+import { colors } from "../lib/graphics";
+import { Component } from "geotic";
+
+export default class Animate extends Component {
+  static properties = {
+    startTime: null,
+    duration: 100,
+    animation: {
+      type: "color",
+      stops: [colors.defaultColor],
+    },
+  };
+
+  onSetStartTime(evt) {
+    this.startTime = evt.data.time;
+    evt.handle();
+  }
+
+  onRemove() {
```

Comment:
there are a couple built in callbacks that might be useful here:

```js
onDetached()
onDestroyed()
```

https://github.com/ddmills/geotic#components

Reply:
It though onRemove was a builtin?

Reply:
Nope doesn't look like it! How does this even work?? 🤯 

Thanks for the catch 😄